.github/workflows/build-push-tool-images.yml

```diff
@@ -0,0 +1,66 @@
+name: Build & Push Tool Image to GHCR
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+    paths:
+      - '.github/workflows/build-push-tool-images.yml'
+      - 'src/loaders/compute_tools/**/Dockerfile'
+  push:
+    branches:
+      - main
+      - master
+      - develop
+    paths:
+      - '.github/workflows/build-push-tool-images.yml'
+      - 'src/loaders/compute_tools/**/Dockerfile'
+
+jobs:
+  build-push-tool-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out GitHub Repo
+        uses: actions/checkout@v3
+
+      - name: Get current date
+        id: date
+        run: echo "date=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_OUTPUT
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: '${{ secrets.GHCR_USERNAME }}'
+          password: '${{ secrets.GHCR_TOKEN }}'
+
+      - name: Build and push tool images
+        id: build-and-push
+        run: |
+          tools=(
+            mash
+            gtdb_tk
+            checkm2
+            microtrait)
+
+          for tool in "${tools[@]}"; do
+            dockerfile="./src/loaders/compute_tools/$tool/Dockerfile"
+            tag="ghcr.io/${{ github.event.repository.owner.login }}/${{ github.event.repository.name }}:$tool"
+
+            echo "Building and pushing $tool image..."
+            echo "tagging image with $tag"
+
+            docker buildx build --file "$dockerfile" --tag "$tag" \
+              --build-arg "BUILD_DATE=${{ steps.date.outputs.date }}" \
+              --build-arg "VCS_REF=${{ github.sha }}" \
+              --build-arg "BRANCH=${{ github.ref }}" \
+              --build-arg "TAG=${{ github.ref }}" \
```

Comment:
Should this be $tag or is github.ref intentional?

Reply:
hmmm I had the same question. But I copied this from the template. Tag does seem to be overwritten by explicitly defined `tags`. So I didn't change anything just to be safe. 
https://github.com/kbase/.github/blob/main/.github/workflows/reusable_build-push.yml#L57-L60

Reply:
Fair enough

:+1: